biz.aQute.bndlib.tests/bnd.bnd

```diff
@@ -4,10 +4,11 @@
 -buildpath: \
     biz.aQute.bndlib;version=project,\
     biz.aQute.bnd.test;version=project,\
+	org.osgi.service.component;version=6.0.0,\
```

Comment:
Why is this version 6? In the OSGi build it is version 1.3.0. We need to keep in sync with the version number.